### PR TITLE
🌱 Update basic e2e trigger on README.md

### PR DESCRIPTION
**Pull request by mboukhalfa**
Update basic e2e trigger after migrating to prow

### CODE REVIEW

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Comment by tuminoid** [comment]
/override test-centos-e2e-integration-main test-ubuntu-integration-main
Its a readme file.

**Review by tuminoid — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -164,10 +164,12 @@ Release-1.4 branch:
...
 Unlike integration tests, basic tests focus on the target cluster creation
-without involving pivoting from the bootstrap cluster. To run basic tests,
-replace `integration` with `basic` for instance:
+without involving pivoting from the bootstrap cluster. To run basic tests use:
 
-- **/test-ubuntu-e2e-basic-main** runs basic e2e tests with main branch on Ubuntu
+- **/test metal3-ubuntu-e2e-basic-test-main** runs basic e2e tests with main
+ branch on Ubuntu
+- **/test metal3-centos-e2e-basic-test-release-1-6** runs basic e2e tests on
+ release-1.6 branch with centos
```
Comment (furkatgofurov7) [inline README.md:172]:
```suggestion
- **/test metal3-centos-e2e-basic-test-release-1-6** runs basic e2e tests with
 release-1.6 branch on CentOS
```

**Review by furkatgofurov7 — COMMENTED** [review]
/approve

**Pull request merged**